ui/src/containers/ArtilleryReport.svelte

```diff
@@ -84,60 +81,53 @@
       {#await promise}
         <LoadingFlat />
       {:then data}
-        <Breadcrumbs
-          displayMode="Artillery Load Test" />
-        <br />
-
-        <CardSummary value={data.summary} />
-
-        <ArtilleryDetailsCard
-          build={data ? data : {}}
-          on:artilleryThreshold={() => showArtilleryThreshold(data.summary, $userSession$)} />
-
-        <Tabs build={data ? data : {}} displayMode="artillery" />
-
-        {#if data.summary.latencyP95 !== undefined}
-          <div class="my-4">
-            <div class="float-right">
-              <button
-                on:click={download}
-                title="Download JSON"
-                class="bg-gray-300 hover:bg-gray-400 textdark font-bold py-1 px-1
-              rounded-lg inline-flex items-center">
-                <Icon cssClass="">
-                  <path
-                    d="M4 16v1a3 3 0 003 3h10a3 3 0 003-3v-1m-4-4l-4 4m0 0l-4-4m4 4V4" />
-                </Icon>
-              </button>
+        <BuildDetailsSlot
+          {data}
+          componentType="artillery"
```

Comment:
Can we have these display with uppercase casing to match the other breadcrumbs parts? (i.e. "Artillery", "URL", "Lighthouse" etc)

<img width="331" alt="Screenshot 2023-09-28 at 3 09 09 pm" src="https://github.com/SSWConsulting/SSW.CodeAuditor/assets/11418832/4fb00d51-a678-486a-a856-09849ac22cd2">

Reply:
Done - display uppercase in breadcrumb for consistency

<img width="295" alt="image" src="https://github.com/SSWConsulting/SSW.CodeAuditor/assets/67776356/85cf0092-508e-40a0-9284-118ee45b81e7">